File: TreatUs/Client/src/app/models/SelectedRestaurant.ts
import { Menu } from "./Menu";
export class SelectedRestaurant {
    id : string;
    restaurantName: string;  
    address:string;
    telephoneNo:string;
    category:string;
    menus: Menu[];
              
    public constructor(init?: Partial<SelectedRestaurant>) {
        Object.assign(this, init);
    }

 }

File: TreatUs/Treat_Api/Models/SelectedRestaurant.cs
using System.Collections.Generic;


namespace Treat_Api.Models
{
    public class SelectedRestaurant
    {
        public string Id { get; set; }

        public string RestaurantName { get; set; }

        public string Address { get; set; }

        public string TelephoneNo { get; set; }

        public string Category { get; set; }

        public List<Menus> Menus { get; set; }
        
    }
}
